src/main/scala/com/databricks/labs/overwatch/pipeline/Migration.scala

```diff
@@ -0,0 +1,184 @@
+package com.databricks.labs.overwatch.pipeline
+
+import com.databricks.dbutils_v1.DBUtilsHolder.dbutils
+import com.databricks.labs.overwatch.env.{Database, Workspace}
+import com.databricks.labs.overwatch.utils._
+import org.apache.log4j.{Level, Logger}
+import com.databricks.labs.overwatch.utils.Helpers.removeTrailingSlashes
+
+import java.io.FileNotFoundException
+
+
+class Migration(_sourceETLDB: String, _targetPrefix: String, _workspace: Workspace, _database: Database, _config: Config)
```

Comment:
add comments

This comment is for the entire PR -- all classes / funcs need code comments. Class level comments should detail the purpose / intent of the class in English and funcs should do the same for all func parameters and the intent of the function